examples/events/README.md

```diff
@@ -221,8 +46,10 @@ function Parent(props) {
 }
 
 function Child(props) {
-  const dropzoneProps = useMemo(() => computeDropzoneProps(props, 'child'), [
-    props.preventDrag,
+  const dropzoneProps = useMemo(() => ({
+    ...computeDropzoneProps(props, 'child'),
+    noDragEventsBubbling: true
+  }), [
```

Comment:
This is the simplest use of the new options in the README.md. Will help at least 90% of users that read this. Had to grok a lot of this to realize the `noClick` option is analogous to this.

Reply:
Will try to update the other examples to match this one.